src/dynarec/rv64/dynarec_rv64_660f_vector.c

```diff
@@ -1096,24 +1124,53 @@ uintptr_t dynarec64_660F_vector(dynarec_rv64_t* dyn, uintptr_t addr, uintptr_t i
             }
             break;
         case 0x70:
-            if (rv64_xtheadvector) { DEFAULT_VECTOR; } // lack of vrgatherei16.vv
-
             INST_NAME("PSHUFD Gx, Ex, Ib");
             nextop = F8;
             SET_ELEMENT_WIDTH(x1, VECTOR_SEW64, 1);
-            GETGX_vector(q0, 1, VECTOR_SEW64);
+            GETGX_empty_vector(q0);
```

Comment:
epty vector must be gathered AFTER all other vectors, incase GX==EX!

Reply:
Oops, indeed!